include/stdlib.h

```diff
@@ -128,6 +128,19 @@ extern "C"
 
 void      srand(unsigned int seed);
 int       rand(void);
+void      lcong48(FAR unsigned short param[7]);
```

Comment:
Let's use prototypes from POSIX:
```
double drand48(void);
double erand48(unsigned short int xsubi[3]);
long int jrand48(unsigned short int xsubi[3]);
void lcong48(unsigned short int param[7]);
long int lrand48(void);
long int mrand48(void);
long int nrand48(unsigned short int xsubi[3]);
unsigned short int *seed48(unsigned short int seed16v[3]);
void srand48(long int seedval);
```
I mean add `int` here and in implementation files. Not a big deal, but still let's do it.

Reply:
Yes, you are right.

Reply:
Done!